lib/suites/LinkedDataSignature.js

```diff
@@ -278,7 +278,7 @@ module.exports = class LinkedDataSignature extends LinkedDataProof {
     }
 
     // ensure verification method has not been revoked
-    if(framed.revoked !== undefined) {
+    if(framed.revoked) {
```

Comment:
Why the change here? I'd prefer to leave it how it was.

Reply:
No problem! Reverting.